oak/server/rust/oak_loader/src/options.rs

```diff
@@ -80,6 +87,22 @@ pub struct Opt {
     config_files: Vec<ConfigEntry>,
 }
 
+#[derive(Debug, serde::Deserialize)]
+#[serde(deny_unknown_fields)]
+struct SignatureManifest {
+    application_name: String,
```

Comment:
What is this used for? I cannot think of anything useful for it. Also, the signatures should be agnostic of the actual application, right? Or are you imagining they work only for a specific application? If only for a specific application, why is the application identified by its name only? It does not seem to offer any security guarantee